tests/functions_tests/test_poolings.py

```diff
@@ -10,31 +10,32 @@
 from onnx_chainer.testing import test_onnxruntime
 
 
-@testing.parameterize(
-    {'name': 'average_pooling_2d', 'ops': F.average_pooling_2d,
-     'in_shape': (1, 3, 6, 6), 'args': [2, 1, 1]},
-    {'name': 'average_pooling_nd', 'ops': F.average_pooling_nd,
-     'in_shape': (1, 3, 6, 6, 6), 'args': [2, 1, 1]},
-    {'name': 'max_pooling_2d', 'ops': F.max_pooling_2d,
-     'in_shape': (1, 3, 6, 6), 'args': [2, 1, 1]},
-    {'name': 'max_pooling_nd', 'ops': F.max_pooling_nd,
-     'in_shape': (1, 3, 6, 6, 6), 'args': [2, 1, 1]},
-)
+@testing.parameterize(*(testing.product({
+    'name': ['max_pooling_2d'],
+    'in_shape': [(1, 3, 6, 5)],
+    'args': [[2, 2, 0]],
+    'cover_all': [False, True],
+}) + testing.product({
+    'name': ['average_pooling_2d'],
+    'in_shape': [(1, 3, 6, 5)],
+    'args': [[2, 2, 0]],
+    'cover_all': [None],
+}) + testing.product({
+    'name': ['max_pooling_nd'],
+    'in_shape': [(1, 3, 6, 5, 4)],
+    'args': [[2, 2, 0]],
+    'cover_all': [False, True],
+}) + testing.product({
+    'name': ['average_pooling_nd'],
+    'in_shape': [(1, 3, 6, 5, 4)],
+    'args': [[2, 2, 0]],
+    'cover_all': [None],
+})))
```

Comment:
Could you give `pad=1` to these test cases instead of `pad=0`?
And could you add another test case to check the behavior when stride > ksize? like:
```
testing.product({
    'name': ['max_pooling_2d'],
    'in_shape': [(1, 3, 6, 5)],
    'args': [[2, 3, 1]],
    'cover_all': [True],
})))
```
and please raise a runtime error when stride larger than ksize is given to the converters.